virttest/remote.py

```diff
@@ -1442,3 +1455,31 @@ def run_command(self, vm_ip, command, vm_user="root", runner=None,
             raise exceptions.TestFail("Failed to run '%s' in the VM: %s",
                                       cmd, detail)
         return ret
+
+    def cmd_output_safe(self, cmd, timeout=CMD_TIMEOUT):
+        """
+        Unify the interface for session.cmd_output_safe()
+        :param cmd: The command to execute
+        :param timeout: Ignored so far
+        :return: cmd_result.stdout_text
+        """
+        return self.run_command(cmd).stdout_text.strip()
```

Comment:
I assume timeout is not used actually

Reply:
@chunfuwen , yes. I have written it in the docstring. This is to align with the API for session. Because I want remote.VMManager object can be treated as a session object, I have to create these adapting functions to align with session APIs.

Reply:
I assume ssh support specifying timeout, can we pass parameter timeout to ssh as one option in run_command method?
ssh -o ConnectTimeout=10 user@remotehost

Reply:
@chunfuwen , good catch. I agree it is better to utilize the timeout parameter. But the 'timeout' is for command execution, and your suggestion is for ssh connection timeout. So it seems they are different.  Is it good to mess them up?  What do you think of?

Reply:
@dzhengfy ,go ahead for your solution with timeout in command

Reply:
@chunfuwen I updated according to your suggestion to pass 'timeout'. Thanks.